README.md

```diff
@@ -133,3 +133,23 @@ to restart Synapse to install the plugin.
 ## Development
 
 TODO. It's a TypeScript project with a linter.
+
+### Development and testing with mx-tester
+
+WARNING: mx-tester is currently work in progress, but it can still save you some time and is better than struggling with nothing.
+
+If you have docker installed you can quickly get setup with a development environment by using
```

Comment:
You should probably give everything as a list of commands.

e.g.

> 
> Installing mx-tester
> 
> ```sh
> $ curl ...   # To install Rust
> $ cargo install mx-tester
> ```

> Build an image with `synapse_antispam`:
> 
> ```sh
> $ mx-tester build
> ```

etc.